### PR TITLE
feat: add certification date prop to project

### PR DESCRIPTION
**Pull request by lialila**
Story: https://trello.com/c/6EkhLqjX/522-project-status-on-portfolio-pages-is-fetched-from-fpm-5-13
Preview: https://storybook-4nb9e5ipf-treely.vercel.app/?path=/story/slices-projectsgrid--minimal

### CODE REVIEW

File: src/slices/ProjectGridCard/ProjectGridCard.tsx [hunk 1]
```diff
@@ -49,6 +49,16 @@ export const ProjectGridCard = ({
               {project.location}
             </Text>
           </Tag>
+          {project.certificationDate && (
```
Comment (lukasbals) [inline src/slices/ProjectGridCard/ProjectGridCard.tsx:52]: If there is no certification date we should add something like "Certification in progress". Ok?

**Review by lukasbals — COMMENTED** [review]
One comment. Otherwise great work!

**Review by lukasbals — COMMENTED** [review]
Another question: Is the `footerSubTitle` already removed everywhere?

**Comment by lialila** [comment]
> Another question: Is the `footerSubTitle` already removed everywhere?

Checked: not everywhere, I'll remove it

File: src/slices/ProjectFacts/messages.de.ts [hunk 2]
```diff
@@ -1,5 +1,4 @@
 const messagesDe = {
   'sections.projectFacts.projectInfo.value': 'Projekt Infos',
-  'projects.projectFacts.properties.area': 'Fläche',
```
Comment (lialila) [inline src/slices/ProjectFacts/messages.de.ts:3]: the naming is wrong, also I didn't find any use case of this message, so I'm removing it

**Review by lukasbals — APPROVED** [review]

**Review by lukasbals — APPROVED** [review]

**Pull request merged**